packages/fluentui/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -402,30 +404,41 @@ const menuItemStyles: ComponentSlotStylesPrepared<MenuItemPropsAndState, MenuVar
 
   menu: ({ variables: v }) => ({ zIndex: v.menuZIndex }),
 
-  indicator: ({ props: p, variables: v, rtl }) => ({
-    position: 'relative',
-    float: 'right',
-    left: pxToRem(12),
-    userSelect: 'none',
-    marginRight: pxToRem(4),
+  indicator: ({ props: p, variables: v, rtl }) => {
+    return {
+      position: 'relative',
+      float: 'right',
+      left: pxToRem(12),
+      userSelect: 'none',
+      marginRight: pxToRem(4),
+
+      ...(p.inSubmenu && {
+        position: 'absolute',
+        top: pxToRem(6),
+        right: pxToRem(2),
+        left: 'unset'
+      }),
 
-    ...(p.inSubmenu && {
-      position: 'absolute',
-      top: pxToRem(6),
-      right: pxToRem(2),
-      left: 'unset'
-    }),
+      ...(rtl && {
+        transform: `scaleX(-1)`
+      }),
+      content: '" "',
+      display: 'block',
+      overflow: 'hidden',
+      height: pxToRem(16),
+      width: pxToRem(16),
 
-    backgroundImage: submenuIndicatorUrl(v.color, p.vertical),
-    ...(rtl && {
-      transform: `scaleX(-1)`
-    }),
-    content: '" "',
-    display: 'block',
-    overflow: 'hidden',
-    height: pxToRem(16),
-    width: pxToRem(16)
-  })
+      backgroundImage: submenuIndicatorUrl(v.indicatorColor, p.vertical),
+
+      ...(p.active && {
+        backgroundImage: submenuIndicatorUrl(v.activeIndicatorColor, p.vertical),
+        ...(p.primary && { backgroundImage: submenuIndicatorUrl(v.activePrimaryIndicatorColor, p.vertical) })
```

Comment:
One potential regression could be this: 
![image](https://user-images.githubusercontent.com/4512430/76695824-f1b2d780-6683-11ea-89f9-3bcbd9bcd0a0.png)

The same is for the non-vertical primary menu. Not sure if these are valid variants for teams theme, but I noticed them, so thought I should report :)

Reply:
Good catch, I expected that it will be catched by Screener. Will check is there a visual test for it, if no, will add it before merging this 👍